features/features.go

```diff
@@ -124,6 +126,8 @@ var featureNames = map[SnapdFeature]string{
 	ConfdbControl: "confdb-control",
 
 	AppArmorPrompting: "apparmor-prompting",
+
+	GPIOChardevSupport: "gpio-chardev-support",
```

Comment:
other options could be `gpio-chardev-interface{,-support}`, but I'm fine with the current flag name too

@pedronis do yo have any suggestions here?